### DISCUSSION TITLE
The origin

### DISCUSSION THREAD

**Discussion by MustafaTayil** [wenbinyugroup/.github#5] [SwiftComp Related Discussions] [answered]
Hi,
I am currently calculating the stiffness matrix (either the Euler or Timoshenko beam) for a 2D I-beam section. Could you please clarify the reference origin that SwiftComp uses when determining the sectional properties? Specifically, does SwiftComp calculate these properties with respect to the centroid of the I-beam shape? 
Please note that I am selecting the I-beam section from the "Common SG" options, not drawing the shape manually.
Thanks

**Comment by unitasium** [comment]
The properties are calculated with respect to the origin of the cross-sectional geometry.

**Reply by MustafaTayil** [reply]
Is the origin of the I beam that I can select from the option (Common SG) the centroid?

**Reply by unitasium** [reply, marked as answer]
If you are asking about the Abaqus-SwiftComp GUI, I don't think there is such an option. If you know the coordinates of the centroid, then you can manually move the cross-section in CAE so that the origin coincides with the centroid.

**Reply by MustafaTayil** [reply]
Thanks sir for your help
